mysys/mulalloc.c

```diff
@@ -90,11 +93,15 @@ void *my_multi_malloc_large(PSI_memory_key key, myf myFlags, ...)
     length=va_arg(args,ulonglong);
     tot_length+=ALIGN_SIZE(length);
   }
+  ret_total_length= va_arg(args, size_t *);
```

Comment:
Add return length as an explicit argument to the function and not a var_arg parameter.
This allows the compiler to check the type of variable and also that one has not forgot to add it.